tests/nn/util_test.py

```diff
@@ -595,19 +595,18 @@ def test_add_sentence_boundary_token_ids_handles_3D_input(self):
     def test_remove_sentence_boundaries(self):
         tensor = Variable(torch.from_numpy(numpy.random.rand(3, 5, 7)))
         mask = Variable(torch.from_numpy(
-                numpy.array([[1, 1, 1, 0, 0],
```

Comment:
What does this change do?

Reply:
Oh is it testing that it works if you have only `<S>` and `</S>` tokens? If so, can you add that as a comment.

Reply:
Yes, that's it.  Added a comment.